### DISCUSSION TITLE
all-the-icons completion no longer working with Vertico

### DISCUSSION THREAD

**Discussion by catweazle9** [minad/vertico#539] [Q&A]
I'm using the SVG-based `all-the-icons` package https://github.com/domtronn/all-the-icons.el/tree/svg
This was working very nicely until yesterday when I ran `M-x package-upgrade-all` (upgrading Vertico to version 1.9), after which icons no longer appeared in consult-buffer, find-file, etc (although they still show up in dired).

I'm a novice with elisp/edebug but I finally narrowed the problem down to Vertico commit `f958066`
https://github.com/minad/vertico/commit/f9580665ce1b896f1ad6d9c6b7ceb47c748eb073

If I checkout the prior commit `267d4c9` the icons display correctly again.

Is this something that could/should be fixed in Vertico or does the SVG icons package need to be updated?

Thanks, John.

**Comment by minad** [comment]
See https://github.com/iyefrat/all-the-icons-completion/issues/32

I recommend to migrate to nerd-icons-completion.

**Reply by catweazle9** [reply]
Thank you, manually appying https://github.com/iyefrat/all-the-icons-completion/pull/33/commits/3f5282f4ba36ff8154522cd469ac32afe5b3c7f9 fixed the issue.

I've also now migrated to nerd-icons-completion after spending way too much time choosing a Nerd font - finally settled on `Hack Nerd Font` which seems fairly similar to `consolas` on Windows!
